chromium_src/chrome/browser/renderer_context_menu/render_view_context_menu.cc

```diff
@@ -37,6 +41,10 @@
 #include "ui/gfx/paint_vector_icon.h"
 #include "url/origin.h"
 
+#if !BUILDFLAG(IS_ANDROID) && !BUILDFLAG(IS_IOS)
```

Comment:
`IS_IOS` should not be included here, buildflags in cpp must match bulidflags in gn and this is only `!is_android` in gn. ios would never include this code anyway.

Reply:
actually you can drop `IS_ANDROID` as well because render_view_context_menu.cc is already desktop only

Reply:
fixed